howitz.min.toml.example

```diff
@@ -0,0 +1,18 @@
+[flask]
+SECRET_KEY =
+DEBUG = true
+
+[howitz]
+storage = "./howitz.sqlite3"
+devmode = true
```

Comment:
What's `devmode` do?  Doesn't sound very production-y to me...

Reply:
We tried adding a cli-flag `devmode` to make it very explicit when running for development: debug is set to True as well as some other things. But we stopped trying to support it as a cli-flag and now keep it only in the config-file because making it work with `flask --app howitz run` was proving too hard/not worth the effort.